### PR TITLE
Add fix for Snowflake cacert issue

### PR DESCRIPTION
**Pull request by blairj09**
There's a current issue with the Snowflake driver that prevents it from working properly without first modifying `/opt/rstudio-drivers/snowflake/bin/lib/rstudio.snowflakeodbc.ini`. This PR introduces a fix for the issue until it is resolved upstream.

### CODE REVIEW

**Comment by blairj09** [comment]
Confirmed fix:

File: Dockerfile_multi [hunk 1]
```diff
@@ -268,6 +268,8 @@ RUN curl -O https://drivers.rstudio.org/7C152C12/installer/rstudio-drivers_${DRI
     cat /opt/rstudio-drivers/odbcinst.ini.sample | tee /etc/odbcinst.ini && \
     # Fix odbcinst.ini until we release 1.6.1 .deb/.rpm
     sed -i '/Installer = RStudio$/cInstaller = RStudio Pro Drivers' /etc/odbcinst.ini
```
Comment (colearendt) [inline Dockerfile_multi:270]: missing a line-extension here
```suggestion
    sed -i '/Installer = RStudio$/cInstaller = RStudio Pro Drivers' /etc/odbcinst.ini && \
```

**Review by colearendt — CHANGES_REQUESTED** [review]
Just the one issue! Thanks for fixing this!

File: Dockerfile [hunk 2]
```diff
@@ -250,7 +250,9 @@ RUN curl -O https://drivers.rstudio.org/7C152C12/installer/rstudio-drivers_${DRI
     rm rstudio-drivers_${DRIVERS_VERSION}_amd64.deb && \
     cat /opt/rstudio-drivers/odbcinst.ini.sample | tee /etc/odbcinst.ini && \
     # Fix odbcinst.ini until we release 1.6.1 .deb/.rpm
-    sed -i '/Installer = RStudio$/cInstaller = RStudio Pro Drivers' /etc/odbcinst.ini
+    sed -i '/Installer = RStudio$/cInstaller = RStudio Pro Drivers' /etc/odbcinst.ini && \
```
Comment (colearendt) [inline Dockerfile:253]: Actually, do we still need this?

**Review by colearendt — APPROVED** [review]
🎉

**Pull request merged**